torchao/dtypes/uintx/int4_xpu_layout.py

```diff
@@ -429,15 +428,14 @@ def get_plain(self) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor]:
             # TODO: move this to `unpack_tinygemm_scales_and_zeros`?
             scale = scale.reshape(scale.shape[:-1]).contiguous()
             zero = zero.reshape(zero.shape[:-1]).contiguous()
-            int_data = quantize_affine(
+            int_data = quantize_affine_float_zero_point(
```

Comment:
this is actually specific to fbgemm I think, we'd need to rename in a future PR cc @jainapurva